spec/codecs/compress_spooler_spec.rb

```diff
@@ -143,6 +143,33 @@
           expect(buffer.size).to eq(2)
         end
       end
+
+      context "message spooling if min flush time is set" do
+        let(:spool_size) { 10 }
+        min_flush_time = 2
+        let(:min_flush_time) { min_flush_time }
+        subject(:codec) { LogStash::Codecs::CompressSpooler.new("spool_size" => spool_size, "min_flush_time" => min_flush_time) }
```

Comment:
I would recommend moving the constructor parameters to their own let, so we could make this line smaller. Also using ```described_class``` (see https://www.relishapp.com/rspec/rspec-core/docs/metadata/described-class) will help on redability.